opnode/rollup/sync/start.go

```diff
@@ -1,3 +1,33 @@
+// The sync package is responsible for reconciling L1 and L2.
+//
+//
+// The ethereum chain is a DAG of blocks with the root block being the genesis block.
+// At any given time, the head (or tip) of the chain can change if an offshoot of the chain
+// has a higher number. This is known as a re-organization of the canonical chain.
+// Each block points to a parent block and the node is responsible for deciding which block is the head
+// and thus the mapping from block number to canonical block.
+//
+// The optimism chain has similar properties, but also retains references to the ethereum chain.
+// Each optimism block retains a reference to an ethereum (L1) block and to it's parent optimism (L2) block.
```

Comment:
s/it's/its

Reply:
also I'd just say "L1 block"  and "L2 block" as we do pretty much everywhere else